tests/scripts/booking/handle_expired_bookings_test.py

```diff
@@ -176,11 +177,31 @@ def should_notify_of_todays_expired_bookings(self, mocked_send_email_recap, app)
             [expired_today_cd_booking],
         )
 
+    @mock.patch("pcapi.scripts.booking.handle_expired_bookings.send_expired_bookings_recap_email_to_beneficiary")
+    def test_should_not_notify_of_todays_expired_eac_bookings(self, mocked_send_email_recap, app) -> None:
+        # Given
+        now = datetime.utcnow()
+        long_ago = now - timedelta(days=31)
+        dvd = ProductFactory(subcategoryId=subcategories.SUPPORT_PHYSIQUE_FILM.id)
+        EducationalBookingFactory(
+            stock__offer__product=dvd,
+            dateCreated=long_ago,
+            isCancelled=True,
+            status=BookingStatus.CANCELLED,
+            cancellationReason=BookingCancellationReasons.EXPIRED,
+        )
+
+        # When
+        handle_expired_bookings.notify_users_of_expired_non_eac_bookings()
+
+        # Then
+        assert not mocked_send_email_recap.called
+
 
 @pytest.mark.usefixtures("db_session")
 class NotifyOfferersOfExpiredBookingsTest:
     @mock.patch("pcapi.scripts.booking.handle_expired_bookings.send_expired_bookings_recap_email_to_offerer")
-    def should_notify_of_todays_expired_bookings(self, mocked_send_email_recap, app) -> None:
+    def test_should_notify_of_todays_expired_bookings(self, mocked_send_email_recap, app) -> None:
```

Comment:
```suggestion
    def test_notify_of_todays_expired_bookings(self, mocked_send_email_recap, app) -> None:
```
Pas de mention que ce ne sont que les individuels ?